telethon_generator/data/errors.csv

```diff
@@ -46,21 +46,21 @@ CALL_OCCUPY_FAILED,500,The call failed because the user is already making anothe
 CALL_PEER_INVALID,400,The provided call peer object is invalid
 CALL_PROTOCOL_FLAGS_INVALID,400,Call protocol flags invalid
 CDN_METHOD_INVALID,400,This method cannot be invoked on a CDN server. Refer to https://core.telegram.org/cdn#schema for available methods
-CHANNELS_ADMIN_PUBLIC_TOO_MUCH,400,"You're admin of too many public channels, make some channels private to change the username of this channel"
+CHANNELS_ADMIN_PUBLIC_TOO_MUCH,400,You're admin of too many public channels, make some channels private to change the username of this channel
 CHANNELS_TOO_MUCH,400,You have joined too many channels/supergroups
-CHANNEL_INVALID,400,"Invalid channel object. Make sure to pass the right types, for instance making sure that the request is designed for channels or otherwise look for a different one more suited"
+CHANNEL_INVALID,400,Invalid channel object. Make sure to pass the right types, for instance making sure that the request is designed for channels or otherwise look for a different one more suited
 CHANNEL_PRIVATE,400,The channel specified is private and you lack permission to access it. Another reason may be that you were banned from it
 CHANNEL_PUBLIC_GROUP_NA,403,channel/supergroup not available
 CHAT_ABOUT_NOT_MODIFIED,400,About text has not changed
 CHAT_ABOUT_TOO_LONG,400,Chat about too long
 CHAT_ADMIN_INVITE_REQUIRED,403,You do not have the rights to do this
-CHAT_ADMIN_REQUIRED,400 403,"Chat admin privileges are required to do that in the specified chat (for example, to send a message in a channel which is not yours), or invalid permissions used for the channel or group"
-CHAT_FORBIDDEN,,You cannot write in this chat
+CHAT_ADMIN_REQUIRED,400,Chat admin privileges are required to do that in the specified chat (for example, to send a message in a channel which is not yours), or invalid permissions used for the channel or group
```

Comment:
Are you sure this error is not a 403?

Reply:
Yes, I'm sure
```
{
	"_": "rpc_error",
	"error_code": 400,
	"error_message": "CHAT_ADMIN_REQUIRED"
}
```

Looks like someone is moved accidentally code `403 `from the `CHAT_FORBIDDEN` to the `CHAT_ADMIN_REQUIRED`